tf_world_frame: "map"
pub_pose_topic_name: "pose_uwb"

sub_uwb_tag_1_topic_name: "uwb/tag_1_serial_ranging"
sub_uwb_tag_2_topic_name: "uwb/tag_2_serial_ranging"

# uwb_tag_1_id: '0'
# uwb_tag_2_id: '1'

uwb_tag_1_id: '1DB8'
uwb_tag_2_id: 'DA36'

# tag_1_x: -0.22
# tag_1_y: 0.0

# tag_2_x: 0.22
# tag_2_y: 0.0

tag_1_x: -0.015
tag_1_y: -0.29 

tag_2_x: -0.015
tag_2_y: 0.29 

# tag_z_height: 0.184
tag_z_height: 0.50

uwb_meas_std: [0.5, 0.5, 0.2] # comes from uwb sensor readings data # [x,y,rotz]
expected_uwb_rate: 10.0